README.md

```diff
@@ -25,6 +25,15 @@ Connect to the `guacamole` web interface at:
 The default credentials are `guacadmin`, `guacadmin` - you should change those
 as soon as possible.
 
+### Volumes ###
+
+#### postgres ####
+
+| Mount Point | Purpose |
+| ----------- | ------- |
+| `dbdata` | Stores all database data for the postgres container |
+| `dbinit` | Stores the postgres initialization script for the guacamole database resources |
```

Comment:
There are probably more of these that should be updated to the [preferred capitalization](https://guacamole.apache.org/):
```suggestion
| `dbinit` | Stores the postgres initialization script for the Guacamole database resources |
```

Reply:
I adjusted this to better reflect references to `guacamole` the container and Guacamole the software in e89866e. Please let me know if you have any feedback on those changes.

Reply:
LGTM!